src/entity/PostReport.ts

```diff
@@ -14,8 +15,8 @@ export class PostReport {
   @Column({ default: () => 'now()' })
   createdAt: Date;
 
-  @Column({ length: 12 })
-  reason: string;
+  @Column({ length: 12, type: 'varchar' })
+  reason: ReportReason;
```

Comment:
Using an `enum` a column, postgresql thinks it is a `text` type. But when you set a max length of the column, it is automatically turned into `varchar`. Using an `enum` with `length` makes the build go crazy. Specifying the type fixes it.

Reply:
👀